purchasing/templates/macros/render_activity_log.html

```diff
@@ -22,9 +22,10 @@
   </div>
 
   {# opportunity #}
-  {%- elif action.action_type == 'opportunity' -%}
+  {%- elif action.action_type == 'post' -%}
   <div class="action-event">
-    An opportunity!
+    {{ pn.print_name(action, current_user) }}
+    <strong>{{ action.action_detail.label }}</strong> the <strong>{{ action.action_detail.title }}</strong> opportunity on {{ action.taken_at|datetimeformat }}. <a href="{{ url_for('opportunities.detail', opportunity_id=action.action_detail.opportunity_id) }}">View it on Beacon</a>.
```

Comment:
@shellicious does this make sense to you? feel free to suggest changes. it will either show up as

"`[You|person]` `[created|updated]` the `[title]` opportunity on `[date]`"